lib/matplotlib/tests/test_agg.py

```diff
@@ -84,7 +84,9 @@ def test_long_path():
     fig.savefig(buff, format='png')
 
 
-@image_comparison(['agg_filter.png'], remove_text=True)
+# # Update style when regenerating the test image
+@image_comparison(['agg_filter.png'], remove_text=True,
+                  style=("classic", "_classic_test_patch"))
```

Comment:
Having this combination is a but bulky given that we write it all over the tests. I suggest to create a `_classic_test` style that hard-coded merges both.

Reply:
@oscargus were you going to update this?  Overall I think this is a great idea...